### PR TITLE
Remember Contexts

### PR DESCRIPTION
**Pull request by colin99d**
Currently if you leave the stocks menu and come back you have to retype GME. That sucks. Now you don't have to. Currently I only changed the stocks menu because I my implementation to be reviewed before I change all of them.
Also, I changed the PATH to be a static variable so that it can be accessed without creating an instance of the class. Figured this could help for other features.

### CODE REVIEW

File: gamestonk_terminal/feature_flags.py [hunk 1]
```diff
@@ -45,3 +45,6 @@
 
 # Enable auto print_help when exiting menus
 ENABLE_EXIT_AUTO_HELP = strtobool(os.getenv("GTFF_ENABLE_EXIT_AUTO_HELP", "False"))
+
+# Remember contexts during session
+REMEMBER_CONTEXTS = strtobool(os.getenv("REMEMBER_CONTEXTS", "True"))
```
Comment (DidierRLopes) [inline gamestonk_terminal/feature_flags.py:50]: On the getenv you want "GTFF_REMEMBER_CONTEXTS"

**Review by DidierRLopes — COMMENTED** [review]
Nice work! I would recommend:
* Doing the `.menu()` outside parent classes, otherwise as a code reader it looks odd that you access a new menu through instantiating a class.

And then once you implement this for all, let's merge it 😄

**Review by DidierRLopes — APPROVED** [review]
@piiq @jmaslek you happy with this as well?

**Review by jmaslek — APPROVED** [review]

**Comment by jmaslek** [comment]
niceee

**Pull request merged**